src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -70,12 +70,13 @@ interface IProps {
   portalData?: IPortalData;
   answerMetadata?: IExportableAnswerMetadata;
   interactiveInfo?: IInteractiveInfo;
+  questionId: string;
 }
 
 export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef((props, ref) => {
   const { url, id, authoredState, initialInteractiveState, legacyLinkedInteractiveState, setInteractiveState, linkedInteractives, report,
     proposedHeight, containerWidth, setNewHint, getFirebaseJWT, getAttachmentUrl, showModal, closeModal,
-    setSupportedFeatures, setSendCustomMessage, setNavigation, iframeTitle, portalData, answerMetadata, interactiveInfo } = props;
+    setSupportedFeatures, setSendCustomMessage, setNavigation, iframeTitle, portalData, answerMetadata, interactiveInfo, questionId } = props;
   const _idNum = parseInt(id, 10);
   const idNum = isFinite(_idNum) ? _idNum : 0;
```

Comment:
I think the questionId prop can go away and you can just use the id property.  The id property is set to the interactiveId which is set to embeddable.ref_id. So it has the same value as the questionId property.

The idNum stuff can go away if we change the id field to be a string and just pass the id prop. See the comments in the LARA PR for more details.